docs/UserGuide.md

```diff
@@ -148,15 +143,22 @@ Format: `edit INDEX [id/PATIENT_HOSPITAL_ID] [n/NAME] [p/PREFERRED_NAME] [f/FOOD
 * Edits the patient at the specified `INDEX`. The index refers to the index number shown in the displayed patient list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* * The `edit` command accepts parameters which consists of:
-    *  `patientHospitalId` integer,
-    * `name`, `preferredName` String with only alphabets character,
+* The `edit` command accepts parameters which consists of:
+    *  `INDEX` positive integer from 1 onwards, indicating the index of patient in the PatientSync list.
+    *  `patientHospitalId` non-negative numeric characters which uniquely identifies patient,
```

Comment:
maybe it might be better to say  non-negative numeric string instead as people might be wondering why not use non-negative integer?

Reply:
Agree!